core/modules/taxonomy/taxonomy.pages.inc

```diff
@@ -61,9 +61,8 @@ function taxonomy_term_page(TaxonomyTerm $term) {
   }
   else {
     $build['no_content'] = array(
-      '#prefix' => '<p>',
+      '#type' => 'help',
       '#markup' => t('There is currently no content classified with this term.'),
-      '#suffix' => '</p>',
```

Comment:
Maybe revert this because it's shown on the front-end to normal users? I'm not sure this is really "help" in the normal sense. If someone were to style `#type = help` elements, would this make sense to show with a help icon?

Reply:
Sure, done.